source/game/kart/KartMove.cc

```diff
@@ -129,67 +170,266 @@ void KartMove::calc() {
         state()->setBoost(false);
     }
 
+    if (state()->isMtBoost()) {
+        if (--m_mtBoostTimer < 1) {
+            state()->setMtBoost(false);
+            m_mtBoostTimer = 0;
+        }
+    }
```

Comment:
We can safely remove this, as this timer and this flag do nothing.

This function is defined at 8058327c and is being inlined.